cognitedata/reveal
# feat(react-components): add search hook for annotation assets in reveal

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-2649

## Description :pencil:
Created a hook to provide annotation asset data in `Reveal`, when the asset is selected in calling application, it will also provide `Image360` & `ImageRevision` data to enter into the selected annotation 360 image station/revision.

## How has this been tested? :mag:

In storybook

## Test instructions :information_source:

- `cd react-components && yarn && yarn build && yarn storybook`
- Select `RevealAnnotationSearchHook` example
- Select an item from the Asset table
- We should enter into the specific 360 Image set and camera should point to the annotation.

## Checklist :ballot_box_with_check:
